src/util/get-all-keys.js

```diff
@@ -0,0 +1,10 @@
+
+export default function keys(obj, enumerable = false) {
+  let listOfKeys = [
+    ...Object[enumerable ? 'getOwnPropertyNames' : 'keys'](obj),
```

Comment:
You are right, sorry I meant mutating the reference ^^. Which I think it's more aligned when using **const** isn't?